scripts/west_commands/runners/core.py

```diff
@@ -139,8 +140,12 @@ def __getitem__(self, item):
     def get(self, option, *args):
         return self.options.get(option, *args)
 
+    def path(self):
```

Comment:
Just a minor thing, but could you initialize an attribute named `path` in the `__init__` method directly, to avoid adding this method?

Something like this in `__init__`:

```py
self.path = os.path.join(self.build_dir, 'zephyr', '.config')
self._parse(self.path)
```

Reply:
> 
> Just a minor thing, but could you initialize an attribute named `path` in the `__init__` method directly, to avoid adding this method?
> 

It would be possible though I want to keep the `path` method as I wanted to somehow, implicitly, suggest that path can only be obtained and is set internally.

Reply:
In Python there is no need for that -- the implementation details can be changed from an attribute to a property later if necessary without users having to change their code.

Reply:
OK, will fix. I am too much from the C++/Java ground where you allow attribute access via getters/setters rather than directly.